README.md

```diff
@@ -6,62 +6,75 @@ PyJulia
 
 Experimenting with developing a better interface to [Julia language](https://julialang.org/) that works with [Python](https://www.python.org/) 2 & 3 and Julia v0.6+.
 
-to run the tests, execute from the toplevel directory
-
-```shell
-tox
-```
-
-See [Testing](#testing) below for details.
-
-**Note** You need to explicitly add julia to your `PATH`, an alias will not work.
-
-`pyjulia` is tested against Python versions 2.7, 3.6, and 3.7.  Older versions of Python (than 2.7)  are not supported.
+PyJulia is tested against Python versions 2.7, 3.6, and 3.7.  Older versions of Python (than 2.7)  are not supported.
 
 Installation
 ------------
+
+**Note:** If you are using Python installed with Ubuntu or `conda`,
+PyJulia does not work by default with standard Python interpreter and
+Julia ≥ 0.7.  For workarounds, see [Troubleshooting](#troubleshooting)
+below.  Same caution applies to other Debian-based and possibly other
+GNU/Linux distributions.
+
 You will need to install PyCall in your existing Julia installation
 
 ```julia
-Pkg.add("PyCall")
+julia> using Pkg  # for julia ≥ 0.7
+julia> Pkg.add("PyCall")
 ```
 
-Your python installation must be able to call Julia.  If your installer
-does not add the Julia binary directory to your `PATH`, you will have to
-add it.
+Your python installation must be able to call command line program
+`julia`.  If your installer does not add the Julia binary directory to
+your `PATH`, you will have to add it.  _An alias will not work._
+
+Then finally you have to install PyJulia.
 
-Then finally you have to install pyjulia.
+**Note:** If you are not familiar with `pip` and have some troubles
+with the following installation steps, we recommend to go through
+[Tutorials in Python Packaging User Guide](https://packaging.python.org/tutorials/).
 
 To get released versions you can use:
 
-```
-pip install julia
+```console
+$ python3 -m pip install --user julia
```

Comment:
I switched `pip` to `python3 -m pip` everywhere since it's a common pitfall that `pip` uses different Python executable than `python`.

Reply:
... and added `--user` since that's likely what they want.